tests/imodels-clients-tests-browser/src/ThumbnailOperations.test.ts

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { assertThumbnail } from "@itwin/imodels-client-test-utils/lib/assertions/BrowserFriendlyAssertions";
+import { ApiOptions, Authorization, AuthorizationCallback, ContentType, DownloadThumbnailParams, IModelScopedOperationParams, IModelsClient, Thumbnail, ThumbnailSize, UploadThumbnailParams } from "@itwin/imodels-client-management";
+import { FrontendTestEnvVariableKeys } from "./setup/FrontendTestEnvVariableKeys";
+
+describe("[Management] ThumbnailOperations", () => {
```

Comment:
Generally I'd say Mocha is BDD-lite, where the way we write test suite and test names in more or less a natural language (e.g. "Thumbnail Operations should download a small thumbnail" reads almost like a real sentence in English).

We do some exceptions to that rule, to make it easier to maintain consistency. For example `"[Management]"` could be a constant to ensure that all tests in this package have it written consistently or using `BackendIModelsAccess.name` instead of `"BackendIModelsAccess"` to refer to a specific class that you are testing and automatically get an error if that class gets renamed. But I don't think those exceptions are applicable here.